components/Containers/DisplayNav.js

```diff
@@ -0,0 +1,8 @@
+import { connect } from 'react-redux';
+import Nav from 'components/Nav/Nav';
+
+const mapStateToProps = state => ({ isXs: state.screenSize.isXs });
+
+const DisplayNav = connect(mapStateToProps)(Nav);
```

Comment:
Instead of this wrapper, you can just make the Nav a connected component.

To avoid testing the redux connection, you can export both the component and the connected component, and just test the component itself.

Reply:
No, I'm saying you dont need to make a separate component to connect.

In other words, delete this file, and just connect the actual Nav function component (you can change it to a class component)

Reply:
You chill with a function component?